heudiconv/convert.py

```diff
@@ -1040,7 +1067,7 @@ def save_converted_files(
         safe_movefile(res_files, outname, overwrite)
         if isdefined(res.outputs.bids):
             try:
-                safe_movefile(res.outputs.bids, outname_bids, overwrite)
+                safe_movefile(bids_files, outname_bids, overwrite)
```

Comment:
would need to double-check on what is the effect here and why e.g. we did not use bids_files to start with here, or why checking `isdefined` for res.output.bids and not bids_files to be defined...